index.html

```diff
@@ -3228,117 +3223,118 @@ <h3>Receiving and parsing content</h3>
                 If |prefix code| is not `0`, then set |record|'s associated
                 <a>bytes</a> to the result of running <a>UTF-8 encode</a> on the
                 prefix <a>string</a> obtained from mapping the value of
-                <var>prefix code</var> to the URL prefix as specified in the
+                |prefix code| to the URL prefix as specified in the
                 [[[NFC-STANDARDS]]] URI Record Type Definition specification,
                 Section 3.2.2.
               </li>
               <li>
-                Let <var>buffer</var> be the <a>byte sequence</a> of <var>
-                ndef</var>'s <a>PAYLOAD field</a> from the second <a>byte</a> to the end.
+                Let |buffer:bytes| be the <a>byte sequence</a> of 
+                |ndef|'s <a>PAYLOAD field</a> from the second <a>byte</a> to the end.
               </li>
               <li>
-                <a data-cite="encoding#concept-stream-push">Push</a> <var>buffer</var>
-                to <var>record</var>'s associated <a>bytes</a>.
+                <a data-cite="encoding#concept-stream-push">Push</a> |buffer|
```

Comment:
🚨 you need to figure out why Encoding didn't export this.

Reply:
Files https://github.com/whatwg/encoding/issues/180.